k2hb_ec2_asg_equality_london.tf

```diff
@@ -0,0 +1,169 @@
+# London
+
+locals {
+  k2hb_equality_london_tags_asg = merge(
+    local.k2hb_equality_tags_asg,
```

Comment:
This local doesn't exist in locals.tf. Can you let me know where it is located please?

Reply:
on the common asg file